api/HTMLElement.json

```diff
@@ -3163,10 +3163,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/ElementCSSInlineStyle/style",
           "support": {
             "chrome": {
-              "version_added": "45"
+              "version_added": "1"
```

Comment:
Ugh, a mixin. Since `MathMLElement` now includes this mixin, we'll need to resolve https://github.com/mdn/browser-compat-data/issues/472. Setting it as supported from Chrome 1 is taking the *any* option from https://github.com/mdn/browser-compat-data/issues/472#issuecomment-696628301.

Reply:
This is the data for HTMLElement in particular, but it's the only place where the data for this mixin is.  Do you think we should keep the changes?

Reply:
Aha. So in BCD we have only api.HTMLElement.style, and https://developer.mozilla.org/en-US/docs/Web/API/ElementCSSInlineStyle/style includes that. That's a weird mismatch, but in any case this BCD entry should be for `HTMLElement` only based on the path, so this is good!